source/containers.tex

```diff
@@ -1989,7 +1989,7 @@
 For associative containers with unique keys the stronger condition holds,
```

Comment:
Change to "holds:" (colon), and add to the previous paragraph: "...is positive, the following condition holds:"